src/QBigInt.cpp

```diff
@@ -27,70 +27,138 @@
 using namespace dev;
 using namespace dev::mix;
 
+void QBigInt::manageException() const
+{
+	try
+	{
+		throw;
+	}
+	catch (boost::exception const& _e)
+	{
+		std::cerr << boost::diagnostic_information(_e);
+	}
+	catch (std::exception const& _e)
+	{
+		std::cerr << _e.what();
+	}
+	catch (...)
+	{
+		std::cerr << boost::current_exception_diagnostic_information();
+	}
+}
+
 QString QBigInt::value() const
 {
-	std::ostringstream s;
-	s << m_internalValue;
-	return QString::fromStdString(s.str());
+	try
+	{
+		std::ostringstream s;
+		s << m_internalValue;
+		return QString::fromStdString(s.str());
+	}
+	catch (...)
+	{
+		manageException();
+		return QString();
+	}
 }
 
 QBigInt* QBigInt::subtract(QBigInt* const& _value) const
 {
-	BigIntVariant toSubtract = _value->internalValue();
-	return new QBigInt(boost::apply_visitor(mix::subtract(), m_internalValue, toSubtract));
+	try
+	{
+		BigIntVariant toSubtract = _value->internalValue();
+		return new QBigInt(boost::apply_visitor(mix::subtract(), m_internalValue, toSubtract));
+	}
+	catch (...)
+	{
+		manageException();
+		return nullptr;
```

Comment:
will add some check in QML for that. could not cause a crash but could throw a JavaScript error.